README.md

```diff
@@ -1 +1,62 @@
-# mag-calculator
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: Bookstore
+
+> Description the project: This project is for Magiciant Calculator. ;)
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update your project title to a matching one(i.e ** Math magicians** and not **Bookstore**. 
Also, it can be better if you improve its description by using the one provided by your **[sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/sneak_peek.md)** to have a meaningful detail about your project.